librz/core/cconfig.c

```diff
@@ -3146,6 +3146,7 @@ RZ_API int rz_core_config_init(RzCore *core) {
 
 	/* prj */
 	SETPREF("prj.file", "", "Path of the currently opened project");
+	SETBPREF("prj.decompress", "true", "Decompress the project file while saving");
```

Comment:
I think it should be `prj.compress` instead, and `Compress the project file` instead